### ISSUE TITLE
How to mass evaluate a simple transformers model on a held-out test set?

### ISSUE THREAD

**Issue by anaconda121** [ThilinaRajapakse/simpletransformers#1256]
**Is your feature request related to a problem? Please describe.**

I'm training a model and would like to evaluate the model on a held-out test set. I have already tuned the hyperparameters using a held-out validation set. Having to do model.predict(["text"]) every single time is too time consuming. 

**Describe the solution you'd like**

Just like there is a model.train_model() and model.eval_model(), can something like model.test_model() exist?

**Describe alternatives you've considered**

1. Do model.eval_model() and eval the model on the test set (sounds reasonable to me now that I think about it)
2. Predict on each sample individually.

**Additional context**

**Comment by anaconda121** [comment]
Seems like Issue #1245 has yielded a solution -- closing for now.
